src/session_impl.cpp

```diff
@@ -6865,6 +6950,7 @@ namespace aux {
 			, struct tracker_response const& resp)
 		{
 			TORRENT_UNUSED(tracker_ips);
+			if (!should_log()) return;
```

Comment:
do you mean surrounding the calls with the macro? all the existence of `tracker_logger` is inside the macro